docs/src/api.md

```diff
@@ -108,7 +108,7 @@ Meshes.SharedVertices
 Meshes.face_connectivity_matrix
 Meshes.vertex_connectivity_matrix
 Meshes.linearindices
-Meshes.lengthscale
+Meshes.element_length_scale
```

Comment:
Sorry, I still think these are not the best names. Can we call them `average_elem_length` and `average_node_distance`?